### PR TITLE
Fill out CI script, add CD script

### PR DESCRIPTION
**Pull request by stwlam**

### CODE REVIEW

File: system.json [hunk 1]
```diff
@@ -587,9 +587,9 @@
...
     "primaryTokenAttribute": "attributes.hp",
-    "url": "https://gitlab.com/hooking/foundry-vtt---pathfinder-2e",
-    "bugs": "https://gitlab.com/hooking/foundry-vtt---pathfinder-2e/-/issues",
-    "changelog": "https://gitlab.com/hooking/foundry-vtt---pathfinder-2e/-/raw/release-v9/CHANGELOG.md",
-    "manifest": "https://gitlab.com/hooking/foundry-vtt---pathfinder-2e/-/jobs/artifacts/release-v9/raw/system.json?job=build",
-    "download": "https://gitlab.com/hooking/foundry-vtt---pathfinder-2e/-/jobs/artifacts/release-v9/raw/pf2e.zip?job=build"
+    "url": "https://github.com/foundryvtt/pf2e",
+    "bugs": "https://github.com/foundryvtt/pf2e/issues",
+    "changelog": "https://github.com/foundryvtt/pf2e/blob/release-v9/CHANGELOG.md",
+    "manifest": "https://github.com/foundryvtt/pf2e/releases/latest/download/system.json",
```
Comment (fryguy1013) [inline system.json:593]: I believe this should be `.../releases/download/latest/system.json`

Comment (stwlam) [reply system.json:593]: https://github.com/stwlam/pf2e/releases/latest/download/system.json

Comment (fryguy1013) [reply system.json:593]: Weird...

https://github.com/fryguy1013/pf2e/releases/latest/download/system.json
https://github.com/fryguy1013/pf2e/releases/download/latest/system.json

**Review by TMunsie — APPROVED** [review]
You may attempt to break at your leisure

**Pull request merged**